src/lib/layers/lane_details/IntersectionMarkings.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
```

Comment:
These files are ported from Leaflet styles: https://github.com/a-b-street/osm2streets/blob/main/street-explorer/js/layers.js

Ben and I have talked about some ideas for changing the output of the rendering layer to be more semantic (describing something as a dashed center line, and letting the API user figure out how to render that). But nothing concrete yet.